tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1893,27 +1893,30 @@ class ParameterizedOpConverterTestBase
 //   how TRT handles the precision inside the TRT network, but should not matter
 //   for the TF -> TRT conversion. Therefore it should be sufficient to test
 //   for FP32.
-class OpConverterTest1 : public ParameterizedOpConverterTestBase {};
+class OpConverterDynamicShape_FP32_Test : 
+  public ParameterizedOpConverterTestBase {};  
+// Base class for tests that need to be tested for both FP32 and FP16.
+class OpConverterDynamicShape_FP32_FP16_Test : 
+  public ParameterizedOpConverterTestBase {};
+// Base class for tests that need to be tested for FP32, FP16, and INT32
+class OpConverterDynamicShape_FP32_FP16_INT8_Test : 
+  public ParameterizedOpConverterTestBase {};
 
-// Instantiate parameter combinations to OpConverterTest1
+// Instantiate parameter combinations to OpConverterDynamicShape_<DT_X...>_Test
 INSTANTIATE_TEST_CASE_P(
-    OpConvTestInstantiation, OpConverterTest1,
+    OpConvTestInstantiation, OpConverterDynamicShape_FP32_Test,
     ::testing::Combine(::testing::ValuesIn(ValidTrtModes),
                        ::testing::Values(DT_FLOAT),
                        ::testing::Values(TrtPrecisionMode::FP32)));
 
-// Base class for tests that need to be tested for both FP32 and FP16.
-class OpConverterTest2 : public ParameterizedOpConverterTestBase {};
 INSTANTIATE_TEST_CASE_P(
-    OpConvTestInstantiation, OpConverterTest2,
+    OpConvTestInstantiation, OpConverterDynamicShape_FP32_FP16_Test,
     ::testing::Combine(::testing::ValuesIn(ValidTrtModes),
                        ::testing::Values(DT_FLOAT, DT_HALF),
                        ::testing::Values(TrtPrecisionMode::FP32)));
 
-// Base class for tests that need to be tested for FP32, FP16, and INT32
-class OpConverterTest3 : public ParameterizedOpConverterTestBase {};
 INSTANTIATE_TEST_CASE_P(
-    OpConvTestInstantiation3, OpConverterTest3,
+    OpConvTestInstantiation, OpConverterDynamicShape_FP32_FP16_INT8_Test,
```

Comment:
This should have suffix `_FP32_FP16_INT32` (see two lines below). This is meant for ops that can process shape (INT32) tensors.

INT8 mode we rarely test with the cpp unit tests. This is because we either need to set quantization ranges (as in #45599) or we would need calibration. Therefore INT8 testing is mostly done by the python integration test.